src/layer/AbstractLayer.ts

```diff
@@ -111,6 +111,13 @@ export class AbstractLayer {
       canvas.height = height;
       const ctx = canvas.getContext('2d');
 
+      const isCanvasValid = await validateCanvasDimensions(canvas);
+      if (!isCanvasValid) {
+        throw new Error(
+          `Image dimensions (${width}x${height}) exceeds the canvas size limit for this browser.`,
```

Comment:
Typo: `exceeds` -> `exceed`

Feel free to resolve.

Reply:
Thanks, fixed.